### ISSUE TITLE
Fresh installation produces an installation error in the Dockerfile npm install steps

### ISSUE THREAD

**Issue by rodobre** [adrgs/requestrepo#16]
Simply cloning the repository. fetching the latest commits with `git pull` and running `docker-compose up -d --force-recreate --build` yields the following error:

```
Building backend
[+] Building 1.9s (7/24)
 => [internal] load build definition from Dockerfile                                   0.0s
 => => transferring dockerfile: 801B                                                   0.0s
 => [internal] load .dockerignore                                                      0.0s
 => => transferring context: 2B                                                        0.0s
 => [internal] load metadata for docker.io/library/python:3.10                         0.7s
 => [ 1/20] FROM docker.io/library/python:3.10@sha256:72e7f78d37f3e3f779eeab9bab71361  0.0s
 => [internal] load build context                                                      0.0s
 => => transferring context: 3.15kB                                                    0.0s
 => CACHED [ 2/20] RUN apt-get update && apt-get install -y     software-properties-c  0.0s
 => ERROR [ 3/20] RUN npm install npm@latest -g &&     npm install n -g &&     n late  1.1s
------
 > [ 3/20] RUN npm install npm@latest -g &&     npm install n -g &&     n latest:
#0 1.061 npm ERR! code EBADENGINE
#0 1.064 npm ERR! engine Unsupported engine
#0 1.065 npm ERR! engine Not compatible with your version of node/npm: npm@10.0.0
#0 1.065 npm ERR! notsup Not compatible with your version of node/npm: npm@10.0.0
#0 1.065 npm ERR! notsup Required: {"node":"^18.17.0 || >=20.5.0"}
#0 1.065 npm ERR! notsup Actual:   {"npm":"9.2.0"}
#0 1.067
#0 1.068 npm ERR! A complete log of this run can be found in:
#0 1.068 npm ERR!     /root/.npm/_logs/2023-09-06T15_46_39_158Z-debug-0.log
------
Dockerfile:6
--------------------
   5 |         nodejs npm
   6 | >>> RUN npm install npm@latest -g && \
   7 | >>>     npm install n -g && \
   8 | >>>     n latest
   9 |
--------------------
ERROR: failed to solve: process "/bin/sh -c npm install npm@latest -g &&     npm install n -g &&     n latest" did not complete successfully: exit code: 1
ERROR: Service 'backend' failed to build : Build failed
```

**Comment by adrgs** [comment]
Fixed in [575117f](https://github.com/adrgs/requestrepo/commit/575117f1cbe99de2047a9dc9b1f98d77035b41fe) by pinning the npm version to 9.2.0 since latest was 10.0.0

**Comment by adrgs** [comment]
Fixed in [575117f](https://github.com/adrgs/requestrepo/commit/575117f1cbe99de2047a9dc9b1f98d77035b41fe) by pinning the npm version to 9.2.0 since latest was 10.0.0
